packages/manager/apps/dedicated/client/app/components/dedicated-cloud/users/activeDirectories/add/federation-add.controller.js

```diff
@@ -0,0 +1,73 @@
+import {
+  FIELD_NAME,
+  GUIDE_FEDERATION,
+  PLACEHOLDER,
+} from './federation-add.constant';
+
+export default class FederationAddCtrl {
+  /* @ngInject */
+  constructor($translate, Alerter, DedicatedCloud) {
+    this.$translate = $translate;
+    this.alerter = Alerter;
+    this.DedicatedCloud = DedicatedCloud;
+  }
+
+  $onInit() {
+    this.FIELD_NAME = FIELD_NAME;
+    this.PLACEHOLDER = PLACEHOLDER;
+
+    this.guideFederationLink =
+      GUIDE_FEDERATION[this.user.ovhSubsidiary] || GUIDE_FEDERATION.DEFAULT;
+
+    this.model = {
+      domainName: null,
+      domainAlias: null,
+      description: null,
+      ip: null,
+      ldapHostname: null,
+      ldapTcpPort: null,
+      sslThumbprint: null,
+      username: null,
```

Comment:
userName ?

Reply:
In the api it is username without capital letter.